src/main/resources/config/config.yaml

```diff
@@ -27,10 +27,19 @@ jerseyClientConfig:
   disabledSecureConnection: ${DISABLE_INTERNAL_HTTPS}
 
 rateLimiter:
-  rate: ${RATE_LIMITER_VALUE:-25}  # the rate is for all requests except POST
-  rateForPost: ${RATE_LIMITER_VALUE_POST:-5}
+  rate: ${RATE_LIMITER_VALUE:-75}  # the rate is for all requests except POST
```

Comment:
is this rate for ALL the nodes? if so it seems wrong. i don't think the service should have information on its environment, in this case the number of nodes. what if we do autoscaling? what if we change desired nodes in aws manually? i think as far as this config is concerned, all it should care about is the rate for ONE node.

Reply:
I was thinking to save on adding more environment variables. I guess this is making it worse. So added new env variable to use for local rate limits. This will give us flexibility to apply local rates using pay infra.